extensions/wikia/PortableInfobox/styles/PortableInfobox.scss

```diff
@@ -81,6 +81,35 @@
 	text-align: center; // center smaller image inside infobox
 }
 
+.pi-image-gallery {
+	position: relative;
+
+	&::after {
+		background-image: url("/extensions/VisualEditor/modules/ve-mw/ui/styles/images/icons/gallery-ltr-white.svg");
+		background-repeat: no-repeat;
+		background-size: contain;
+		content: '';
+		display: block;
+		height: 30px;
```

Comment:
Please consider moving the magic number to a variable for example `$icon-size` and reuse this variable for both `height` and `width`